tools/ou_evidence_provenance.py

```
@@ -0,0 +1,718 @@
+    return {repo_name(path): file_record(path) for path in implementation_paths(study)}
+
+
+def analysis_records(study: str) -> dict[str, dict[str, object]]:
+    paths = [Path(path).resolve() for path in STUDIES[study]["analysis_files"]]
+    return {repo_name(path): file_record(path) for path in sorted(paths, key=repo_name)}
+
+
+def git_available() -> bool:
+    try:
+        completed = subprocess.run(
+            ("git", "rev-parse", "--is-inside-work-tree"), cwd=REPO_ROOT,
```

Ruff (0.16.1)

[error] Starting a process with a partial executable path

(S607)